samples/predict-custom-trained-model.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main(filename, endpointId, project, location = 'us-central1') {
+  // [START aiplatform_predict_custom_trained_model]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.\
+   * (Not necessary if passing values as arguments)
+   */
+
+  // const filename = "YOUR_PREDICTION_FILE_NAME";
+  // const endpointId = "YOUR_ENDPOINT_ID";
+  // const project = 'YOUR_PROJECT_ID';
+  // const location = 'YOUR_PROJECT_LOCATION';
+
+  // Imports the Google Cloud Prediction Service Client library
+  const {PredictionServiceClient} = require('@google-cloud/aiplatform');
+
+  // Specifies the location of the api endpoint
+  const clientOptions = {
+    apiEndpoint: 'us-central1-prediction-aiplatform.googleapis.com',
+  };
+
+  // Instantiates a client
+  const predictionServiceClient = new PredictionServiceClient(clientOptions);
+
+  async function predictCustomTrainedModel() {
+    // Configure the parent resource
+    const endpoint = `projects/${project}/locations/${location}/endpoints/${endpointId}`;
+    const parameters = {
+      structValue: {
+        fields: {},
+      },
+    };
+    const fs = require('fs');
+    const instanceDict = fs.readFileSync(filename, 'utf8');
```

Comment:
We should not use sync methods in sample code.  These often get picked up and used directly in production code by customers, and sync is very, very bad in production.  Instead, please use `util.promisify(fs.readFile)` and then `await` it.